website/docs/docs/collaborate/explore-projects.md

```diff
@@ -57,9 +57,10 @@ To explore the lineage graphs of tests and macros, view [their resource details
     - Refocus on the node and its downstream nodes only
     - Refocus on the node and it upstream nodes only
     - View the node's [resource details](#view-resource-details) page
-- Select a resource to highlight its relationship with other resources in your project. A panel opens on the graph’s right-hand side that displays a high-level summary of the resource’s details. The side panel includes a **General** tab for information like description, materialized type, and other details.
-    - Click the Share icon in the side panel to copy the graph’s link to your clipboard.
-    - Click the View Resource icon in the side panel to [view the resource details](#view-resource-details). 
+- Select a resource to highlight its relationship with other resources in your project. A panel opens on the graph’s right-hand side that displays a high-level summary of the resource’s details. The side panel includes a **General** tab for information like description, materialized type, and other details. In the side panel's upper right corner:
+    - Click the View Resource icon to [view the resource details](#view-resource-details). 
```

Comment:
```suggestion
    - Click the **View Resource icon** to [view the resource details](#view-resource-details). 
```

Reply:
General thought here: We haven't called out these on-screen elements as icons if there is text to select. Should we stick to that format here (and the rest of this doc)?
```suggestion
    - Click **View Resource**  to [view the resource details](#view-resource-details). 
```